### PR TITLE
feat(markdown): add utility to convert markdown to msft component strings

### LINKED ISSUE #289: Migrate FW.Framework markdown-to-react utility
Convert markdown to react-msft components (where available, standard HTML where its not). Port from FW.Framework and create new package in fast-dna named *fast-markdown-react-msft*.

Add build step to create these files.

### PR DESCRIPTION
**Pull request by janechu**
Closes #289

### CODE REVIEW

File: packages/fast-markdown-msft-react/src/index.spec.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    });
+
+    test("should convert a markdown unordered bullet list to a list", function(): void {
+        const markdown: string = "* list item";
+        expect(md.render(markdown).replace(newlineRegex, "")).toBe(
+            "<ul><li><Typography tag=\"p\" typeLevel={2}>list item</Typography></li></ul>"
+        );
+    });
+
+    test("should convert a markdown ordered list to a fluent web list", function(): void {
```
Comment (chrisdholt) [inline packages/fast-markdown-msft-react/src/index.spec.ts]: Probably don't want FW here or throughout

File: packages/fast-markdown-msft-react/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    attrPush: (attributes: string[]) => void;
+}
+
+class FastMarkdownIt {
+    constructor(md: MarkdownIt) {
+        md.core.ruler.push("fast", (new CreateRule(md) as any));
+        md.renderer.rules.paragraph_open = function(): string {
+            return `<Typography tag="p" typeLevel={2}>`;
+        };
+        md.renderer.rules.heading_open = function(tokens: ITokens<IToken>, idx: number): string {
```
Comment (chrisdholt) [inline packages/fast-markdown-msft-react/src/index.ts]: If this is specific to FAST MSFT, should we not use the Heading component? If this isn't specific to MSFT, can you update the ReadMe language?

Comment (chrisdholt) [reply packages/fast-markdown-msft-react/src/index.ts]: Highlighted, it put it where it wanted. I'll keep an eye out for it.

File: packages/fast-markdown-msft-react/src/rule.ts [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+    constructor(md: MarkdownIt) {
+        return (this.replace as any);
+    }
+
+    public replace = (state: IState): void => {
+        this.renderComponents(state);
+    }
+
+    private renderComponents = (state: IState): void => {
```
Comment (nalogiudice) [inline packages/fast-markdown-msft-react/src/rule.ts]: Can you leave a comment somewhere in this file explaining what it is doing?

Comment (janechu) [reply packages/fast-markdown-msft-react/src/rule.ts]: resolved

**Comment by janechu** [comment]
@awentzel its specific to the MSFT exported components, this can only be used if we are importing the MSFT react package because of the differences in markdown between our react packages. For example, there is a Heading in MSFT but not in base

**Pull request merged**